<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisSettingForm</class>
 <widget class="QWidget" name="AnalysisSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="wingListWidget"/>
   </item>
   <item>
    <widget class="QCheckBox" name="discoidalCheckBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cubitalCheckBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startAnalysisPushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
